quickstarts/newrelic-ingest-metric/config.yml

```diff
@@ -0,0 +1,51 @@
+# Name of the quickstart (required)
+name: newrelic-ingest-metric
+
+# Displayed in the UI (required)
+title: New Relic Ingest Metric
+
+# Long-form description of the quickstart (required)
+description: |
+  This quickstart provides a synthetic monitor (and associated dashboard) that will regularly query ingest for each data type (APM Events, Infrastructure Processes, etc.) and post it to the Metric API as a custom metric `newrelic-ingest-metric`. This means that you can query and analyze ingest across your account for 13 months by default, instead of 8 days for events, 30 days for  Logs, etc. The custom metric also includes a facet for each data type, so that you can see, for example, ingest for Mobile Events by appName or Logging by labels.app. The following parameters have default values but are all configurable:
+  - ingest interval (hourly, daily, weekly)
+  - facet for each data type
+  - name of custom metric
+  - addition of custom events
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  Use this script to set up a synthetic monitor that:
+  1. Queries ingestion by data source, faceted by the attribute of your choice (e.g., APM Events by appName)
+  2. Transforms the output into a metric that is pushed to the Metric API
+  3. Can be queried for consumption analysis / dashboards for 13 months
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Authors of the quickstart (required)
+authors:
+  - Nora Shannon Johnson
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - ingest
+  - ingestion
+  - custom metric
+  - synthetic monitor
+  - consumption
+  - billing
+  - facet
+
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+installPlans:
+  - infra-agent-standard
```

Comment:
Looks like the validation is failing and saying this install plan doesn't exist. Did you mean to use the [`infra-agent-targeted`](https://github.com/newrelic/newrelic-quickstarts/blob/main/install/infrastructure/infra-agent-targeted/install.yml#L1) install plan?

Reply:
ooo no I did not, must have been there from the template. do I need to have an installPlan or can I leave it blank @aswanson-nr ?

Reply:
You will need an [install plan](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#install-plans) to invoke the guided install flow in NR1. 

If you intend this to be a documentation only quickstart you can decided to not use an install plan and take the approach similar to this [quickstart](https://github.com/newrelic/newrelic-quickstarts/blob/main/quickstarts/ads/web/ads-web-prebid/config.yml) which only has documentation references. 

Then the user would see something like this in the UI, 


![Screen Shot 2022-02-08 at 10 02 57 AM](https://user-images.githubusercontent.com/1395158/153037878-7c47652e-28c9-4076-a084-a63bfe88f85a.png)